addon/webextension/catcher.js

```diff
@@ -58,6 +58,7 @@ window.catcher = (function () {
       console.error("------Error in promise:", e+"");
       console.error(e.stack);
       exports.unhandled(makeError(e));
+      throw e;
```

Comment:
Can you explain the effect of this? I'm not 100% clear on what'll happen at the top level if we always re-throw. Just an error in the browser console?